tests/js/spec/components/__snapshots__/createProject.spec.jsx.snap

```diff
@@ -1488,7 +1488,7 @@ exports[`CreateProject should deal with incorrect platform name if its provided
                       setReferenceNode={[Function]}
                     >
                       <Container
-                        aria-describedby="tooltip-123456"
+                        aria-describedby=""
```

Comment:
I tried to figure out what was going on here, but was unable to. `jest` has a dependency on `jest-cli@^24` which ends up always resolving to `24.9.0` so I has having a hard time narrowing down to a specific jest version.

The mocks are working and I don't see this behavior in the application itself so seems isolated to jest and our test environment.

Reply:
Should we remove the mock implementation? It seems unused right now.

Reply:
I think we should leave it - the mock still gets called, idk how it gets lost when serializing the component.

Reply:
OK figured this out, it's because tooltip id was being set after component is mounted: https://github.com/getsentry/sentry/pull/15533/files#diff-418d89bac8d87348af39590ddcbe313dL103

Reply:
Nice 👍